.env.example

```
@@ -1,3 +1,3 @@
-SENDINBLUE_API_KEY='xkeysib-3f51c…'
-SMTP_PASSWORD='password'
-GITHUB_TOKEN=ghp_XXXXXXXXX
+OTA_ENGINE_SENDINBLUE_API_KEY='xkeysib-3f51c…'
```

I think it's best to avoid using the abbreviation `OTA`.
```suggestion
OPENTERMSARCHIVE_ENGINE_SENDINBLUE_API_KEY='xkeysib-3f51c…'
```